src/MissionParallelCatchup/parallel_catchup_helm/templates/catchup_workers.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: stellar-core
+spec:
+  clusterIP: None
+  selector:
+    app: stellar-core
+  ports:
+    - port: 11626
+      targetPort: 11626
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: stellar-core
+  labels:
+    app: stellar-core
+spec:
+  serviceName: "stellar-core"
+  replicas: {{ .Values.worker.replicas }}
+  selector:
+    matchLabels:
+      app: stellar-core
+  template:
+    metadata:
+      labels:
+        app: stellar-core
+    spec:
+      containers:
+      - name: stellar-core
+        image: {{ .Values.worker.stellar_core_image }}
+        imagePullPolicy: Always
+        # resource specs copied from the old supercluster mission
+        resources:
+          requests:
+            cpu: "{{ .Values.worker.request_cpu}}"
+            memory: "{{ .Values.worker.request_memory}}"
+            ephemeral-storage: "{{ .Values.worker.request_ephemeral_storage}}"
+          limits:
+            cpu: "{{ .Values.worker.limit_cpu}}"
+            memory: "{{ .Values.worker.limit_memory}}"
+            ephemeral-storage: "{{ .Values.worker.limit_ephemeral_storage}}"
+        command: ["/bin/sh", "/scripts/worker.sh"]
+        ports:
+        - containerPort: 11626
+        env:
+        - name: JOB_QUEUE
+          value: "{{ .Values.redis.job_queue }}"
+        - name: SUCCESS_QUEUE
+          value: "{{ .Values.redis.success_queue }}"
+        - name: FAILED_QUEUE
+          value: "{{ .Values.redis.failed_queue }}"
+        - name: PROGRESS_QUEUE
+          value: "{{ .Values.redis.progress_queue }}"
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: RELEASE_NAME
+          value: "{{ .Release.Name }}"
+        volumeMounts:
+        - name: config
+          mountPath: /config
+        - name: script
+          mountPath: /scripts
+      volumes:
+      - name: config
+        configMap:
+          name: stellar-core-config
+      - name: script
+        configMap:
+          name: worker-script
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: worker-script
+data:
+  worker.sh: |-
+    {{- (.Files.Get "files/worker.sh") | nindent 4 }}
```

Comment:
Oh wow, _indenting the input_ so it becomes a literal in the YAML? Is this really the best that can be done?